opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -401,19 +418,14 @@ def new_stats_exporter(options):
 
     if options.default_monitoring_labels is not None:
         exporter.set_default_labels(options.default_monitoring_labels)
-    else:
-        label = {}
-        key = remove_non_alphanumeric(get_task_value())
-        label[key] = OPENCENSUS_TASK_DESCRIPTION
-        exporter.set_default_labels(label)
     return exporter
 
 
 def get_task_value():
     """ getTaskValue returns a task label value in the format of
-     "py-<pid>@<hostname>".
+     "py-<random_number>@<hostname>".
```

Comment:
Why the removal of the PID? That was deterministic and the way other language exporters work.

Reply:
I agree, ```pid``` makes more sense here.